tests/gui/models/supported-formats.spec.js

```diff
@@ -61,6 +61,32 @@ describe('Browser: SupportedFormats', function() {
 
     describe('.isSupportedImage()', function() {
 
+      _.forEach([
+
+        // Type: 'archive'
+        'path/to/filename.zip',
+        'path/to/filename.etch',
+
+        // Type: 'compressed'
+        'path/to/filename.img.gz',
+        'path/to/filename.img.bz2',
+        'path/to/filename.img.xz',
+
+        // Type: 'image'
+        'path/to/filename.img',
+        'path/to/filename.iso',
+        'path/to/filename.dsk',
+        'path/to/filename.hddimg',
+        'path/to/filename.raw',
+        'path/to/filename.dmg'
```

Comment:
Would it be worth it / possible to check that all these filenames resolve to the correct expected mime-type?
(or is that already covered elsewhere?)

P.S. Looking at https://github.com/resin-io/etcher/pull/1283/files#diff-dd8adcb7744d5236c18272b3e227071eR144 I can now see why the problem that this PR fixes wasn't picked up by the unit-tests - they only check the _first_ of each supported extension type, so of course `.dmg` didn't get tested because it was on the end of the list.